bundler/lib/bundler/templates/newgem/lib/newgem.rb.tt

```diff
@@ -4,6 +4,12 @@ require_relative "<%= File.basename(config[:namespaced_path]) %>/version"
 <%- if config[:ext] -%>
```

Comment:
I am commenting on this line but I mean to comment about line 3. I think line 3's require could become optional with zeitwerk, as zeitwerk would pick up the version file automatically.

So maybe making it conditional with like this might make sense:

```
<%- if config[:zeitwerk] -%>
require_relative "<%= File.basename(config[:namespaced_path]) %>/version"
<%- end -%>
```

Reply:
PS: not sure about line 4 as I am not familiar with gem C/Rust extensions.

Reply:
👍 Changed